dbld/deb

```diff
@@ -16,6 +16,6 @@ if [ ! -d debian ]; then
 	echo "You have no Debian packaging in debian/, this should already by in the tarball at this point"
 	exit 1
 fi
-dpkg-buildpackage -rfakeroot
+dpkg-buildpackage -rfakeroot -d
```

Comment:
I understand this helps with the python dependencies. But this will ignore the non-python dependencies too, won't it? Is it safe to do so?

Reply:
Yes it will, but I think it is safe to do so.

The build dependencies are used by the distribution's build service and there's value in validating them, but their primary purpose is to control builds in the distro.

Now, we could use build-depends, but that would mean that _all_ build dependencies would need to be converted into .deb packages and hosted on OBS or back in the distribution, which would slow us down. I wouldn't want to work on packaging the latest version of pep8/pycodestyle or pylint. And if we don't use them from pip, we would get errors down the path, when users start to use new versions of distros.

So my rationale is:
  * build-dependencies is validated when building in the distribution's context. There we can use an older version of pylint/pycodestyle.
  * we use pip instead of python packages during our private builds, thereby ensuring that we are warning free with the latest/greatest.

Reply:
Ok I see this will not cause problem. Binary api problems will not occur because we do not use <= relation in the versions for now, and we need to build on the same distribution version anyway, that we target the deb packages. And if build dependency is missing, we will see a build error anyway.